components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/listener/ApplicationMgtAuditLogger.java

```diff
@@ -302,6 +304,21 @@ private String buildData(ServiceProvider serviceProvider) {
             data.append("}");
         }
 
+        if (serviceProvider.getTrustedAppMetadata() != null) {
```

Comment:
shall we add this to v2 audit logs as well? V1 audit logs are legacy ones

Reply:
Trusted app metadata is an attribute of the SP object. So this information will anyways be published in the v2 audit log inside the SP object.